sphinxcontrib/confluencebuilder/builder.py

```diff
@@ -17,6 +18,8 @@
 from .writer import ConfluenceWriter
 from docutils import nodes
 from docutils.io import StringOutput
+from docutils.nodes import Node
```

Comment:
Is this import (for `Node`) actually used? I do not see it at the moment, but I may be overlooking it.

Reply:
You are correct. It is not necessary.